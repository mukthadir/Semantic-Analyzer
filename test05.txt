TYPE
   dw : INT;
   ew : dw;
   fw : gw;
   hw : BOOLEAN;
   iw : STRING;
   jw : REAL;

 VAR  
   dy : dw;   ey : ew;   fy : fw;   hy : hw;   iy : iw;   jy : jw;
   ky : w1;   ly : w1;   my : w2;   ny : w2;   oy : w3;

 {
    dy = 1;
    ey = dy + oy;
    WHILE my < ly
    {
       dy = dy+1;
    }

    WHILE my
    {
       dy = dy+1;
    }

    ey = dy;
    fy = 1.0;
    py = qy;
 }

