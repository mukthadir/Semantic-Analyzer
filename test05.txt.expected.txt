INT : dw ew w3 dy ey oy #
REAL : fw jw gw fy jy #
STRING : iw iy #
BOOLEAN : hw w1 w2 hy ky ly my ny #
py : qy #
